app/helpers/dashboard/epic_queues_helper.rb

```diff
@@ -30,12 +30,25 @@ def format_pis(protocol)
     end
   end
 
+  def format_name(epic_queue)
+    identity = epic_queue.identity
+    "#{identity.try(:first_name)} #{identity.try(:last_name)}"
```

Comment:
Identity already has #full_name which does this. So you could just return `epic_queue.identity.full_name`